Firebase/Messaging/FIRMessaging.m

```diff
@@ -37,7 +37,8 @@
 #import "FIRMessagingVersionUtilities.h"
 
 #import <FirebaseCore/FIRReachabilityChecker.h>
-#import <FirebaseInstanceID/FirebaseInstanceID.h>
+
+#import "FirebaseInstanceID_Internal.h"
```

Comment:
Since this is referencing a private header belonging to a different SDK, you'll have to make sure that the header is visible to the FirebaseMessaging build targets via `HEADER_SEARCH_PATHS`.

Referencing the header from a different project will likely break when we try to move to source pods, though, so for the time being you may want to just duplicate the header into Messaging.

Reply:
Yeah I will try to avoid calling internal header in IID, we've used notification to call IID method before, so I will make a change here to calling it through notification, please hold on the review until I notify it's ready.